core/server/src/main/java/alluxio/master/file/meta/InodeTree.java

```diff
@@ -441,29 +652,49 @@ public long reinitializeFile(AlluxioURI path, long blockSizeBytes, long ttl)
    * Returns a list of all descendants of a particular {@link InodeDirectory}. Any directory inode
```

Comment:
I think the javadoc is out-of-date. This method to me will lock a subtree, right?